### PR TITLE
update backend packages

### LINKED ISSUE #278: Upgrade backend packages
**Update following packages:**

- [ ]   djangorestframework==3.9.1
- [ ]   Django==1.11.20
- [ ]   pyyaml==5.1

### PR DESCRIPTION
**Pull request by tasawernawaz**
#### What are the relevant tickets?
Fixes #278 

#### What's this PR do?
- [x] djangorestframework==3.9.1
- [x] Django==1.11.20
- [x] pyyaml==5.1
- [x] pin django-debug-toolbar==1.11 for compatibility

#### How should this be manually tested?

- Site should run without any error.
- Test should pass.

### CODE REVIEW

File: requirements.in [hunk 1]
```diff
@@ -9,7 +9,8 @@ dj-static==0.0.6
 django-redis==4.7.0
 django-server-status==0.3.2
 django-webpack-loader==0.4.1
-djangorestframework==3.5.2
+djangorestframework==3.9.1
+django-debug-toolbar==1.11
```
Comment (noisecapella) [inline requirements.in]: Why is this added here and also below in `test_requirements.txt`?

Comment (tasawernawaz) [reply requirements.in]: @noisecapella newer version of `django-debug-toolbar` was expecting `MIDDLEWARE` in settings instead of `MIDDLEWARE_CLASSES`. For compatibility I have to pin its version here and in test requirements.

Comment (noisecapella) [reply requirements.in]: Why does it need to be here and also in `test_requirements.txt`? Why not just pin it in `test_requirements.txt`?

Comment (tasawernawaz) [reply requirements.in]: @noisecapella My bad, I thought I have to update here too to run project smooth. I have removed this from `requirements.txt`.
